utils/wallpaper.go

```diff
@@ -1,28 +1,19 @@
+// +build !darwin,!linux
```

Comment:
Makes no sense to have this file.

Reply:
Other users might try to build this project. So, I am making sure it builds successfully but doesn't provide the feature.